src/RapidRequest.js

```diff
@@ -88,10 +88,14 @@ const processRequest = async (req) => {
 };
 
 const executeRequest = async (request) => {
-  const context = new Context({
-    ...(request.envVariables || {}),
-    ...(request.testVariables || {}),
-  });
+  const context = new Context(
+    {
+      ...(request.envVariables || {}),
+      ...(request.testVariables || {}),
+    },
+    request.envSecrets // used to mask sensitive context data in the report
```

Comment:
This has no effect in bc the outgoing requests from the RG are reported before the translation even happens so there is no masking needed bc variables are not translated, but this at least brings the executeRequest up to spec with executeTest so that they both create Context objects with a secrets.